InternshipApp/InternshipApp/FlightsApp/ViewModel/FlightViewModel.swift

```diff
@@ -18,17 +18,238 @@ final class FlightViewModel: ObservableObject {
+                    possibleAirports: ["ZRH"]),
+        
+        FlightModel(town: "Rome",
+                    airportCode: "FCO",
+                    airportFullName: "Leonardo da Vinci–Fiumicino Airport",
+                    possibleAirports: ["MUC", "LGA", "ZRH", "SJJ"])
     ]
     
+    @Published var flightOffers: [FlightOfferModel] = [
+        FlightOfferModel(departureCode: "MST",
+                         departureTown: "Mostar",
+                         arrivalCode: "IST",
+                         arrivalTown: "Istanbul",
+                         flightDuration: "12:45",
+                         time: "09:30 AM",
+                         date: Date(),
+                         airCompany: "Qatar Airways",
+                         price: 250),
+        
+        FlightOfferModel(departureCode: "SJJ",
+                         departureTown: "Sarajevo",
+                         arrivalCode: "IST",
+                         arrivalTown: "Istanbul",
+                         flightDuration: "2:15",
+                         time: "01:20 PM",
+                         date: Date(),
+                         airCompany: "Turkish Airlines",
+                         price: 190),
+        
+        FlightOfferModel(departureCode: "MUC",
+                         departureTown: "Munchen",
+                         arrivalCode: "SJJ",
+                         arrivalTown: "Sarajevo",
+                         flightDuration: "1:50",
+                         time: "10:00 AM",
+                         date: Date(),
+                         airCompany: "Lufthansa",
+                         price: 220),
+        
+        FlightOfferModel(departureCode: "LGA",
+                         departureTown: "New York",
+                         arrivalCode: "ZRH",
+                         arrivalTown: "Zurich",
+                         flightDuration: "8:30",
+                         time: "07:00 AM",
+                         date: Date(),
+                         airCompany: "Ryanair",
+                         price: 350),
+        
+        FlightOfferModel(departureCode: "LGA",
+                         departureTown: "New York",
+                         arrivalCode: "ZRH",
+                         arrivalTown: "Zurich",
+                         flightDuration: "8:30",
+                         time: "07:00 AM",
+                         date: Date(),
+                         airCompany: "Swiss Air",
+                         price: 450),
+        
+        FlightOfferModel(departureCode: "FCO",
+                         departureTown: "Rome",
+                         arrivalCode: "LGA",
+                         arrivalTown: "New York",
+                         flightDuration: "9:15",
+                         time: "11:00 AM",
+                         date: Date(),
+                         airCompany: "Alitalia",
+                         price: 380),
+        
+        FlightOfferModel(departureCode: "MUC",
```

Comment:
Please extract this into a separate file, so we don't polute the ViewModel with data stuff :)